src/models/products.Model.ts

```diff
@@ -76,6 +77,10 @@ Product.init(
       type: DataTypes.DATE,
       allowNull: true,
     },
+    available: {
+      type: DataTypes.BOOLEAN,
+      allowNull: true,
```

Comment:
Should we really allow it to be Null? or we should create a defaultValue instead...